### DISCUSSION TITLE
Changing name of Learn AI.

### DISCUSSION THREAD

**Discussion by Gitstar-OC** [fuma-nama/fumadocs#849] [General]
Hey, @fuma-nama, if you could help me could you please tell me that if I should change the name of my project from learn AI to something unique as websites are already crowded with this name of learn ai, something unique may get my seo at the top like for example next js has a unique name and everytime someone searches for it, it goes at top search, similarly for fumadocs, it is the top search. 

Also I have added a sitemap.xml file and submitted it to google search console, What other seo things or tools I should use. 

Also, 
![Screenshot 2024-09-11 174611](https://github.com/user-attachments/assets/2bf9e0c7-98ed-4c6e-9506-10f964a73cd5)

how can I make search results like this ? 
![Screenshot 2024-09-11 174715](https://github.com/user-attachments/assets/c95aacea-2960-4bd0-be97-62ee8bb69067)

Thanks! 
I could not tell you how great help you are providing to me, thanks to the video on your channel of nextjs router about creating site.

**Comment by Gitstar-OC** [comment]
the sitemap is here https://learnai.theme-verse.com/sitemap.xml, and I also have an additonal sitemap.yml file, which I don't know helps or not.

**Comment by fuma-nama** [comment]
It's something to be done with SEO which I can't and won't help you.

**Reply by Gitstar-OC** [reply]
No worries, Just tell me if I should change name as you are experienced in this thing.

**Reply by fuma-nama** [reply]
Just rename it if you wanted, I had renamed Fumadocs from next-docs, it's a good decision

**Comment by Gitstar-OC** [comment]
Great, I am changing the name to **mindect** now! Please give me any kind of important resource about seo where I could learn about it  and develop it properly, I would be also checking out how the seo of top websites are being built like your for example, next js, electron ( specially ), 

### Any help would be great it will just take you half a minute to tell me, but It could provide huge help with that!   

 # ⭕ 
 here's a circle for you👍
